build/helper/metadata_add_all.py

```diff
@@ -237,6 +237,33 @@ def test_add_all_metadata_simple():
                 'description': 'Performs a foo, and performs it well.',
             },
         },
+        'makeAPrivateMethod': {
```

Comment:
Driver functions start with capital letter after the prefix.

Reply:
Given similar name as existed method in metadata_add_all.py(makeAFoo)